rn/Teacher/ios/Teacher/SpeedGrader/SubmissionGrader.swift

```diff
@@ -193,16 +213,15 @@ struct SubmissionGrader: View {
                 fileID = $0
                 snapDrawerTo(.min)
             })
-            Pages(items: GraderTab.allCases, currentIndex: Binding(get: { tab.rawValue }, set: {
-                tab = GraderTab.allCases[$0]
-            })) { tab in
-                VStack(spacing: 0) {
-                    switch tab {
-                    case .grades:
+            GeometryReader { geometry in
+                HStack(spacing: 0) {
+                    VStack(spacing: 0) {
                         SubmissionGrades(assignment: assignment, submission: submission)
                             .clipped()
                         Spacer().frame(height: bottomInset)
-                    case .comments:
+                    }
+                        .frame(width: geometry.size.width, height: geometry.size.height)
```

Comment:
I'm a SwiftUI noob. Help me understand why we need to set the frame on all these tabs to the size of the geometry reader?

Reply:
These are all inside an HStack. Without explicit sizing they will all want to all fit on screen at once. We want each to fit in the outer space and get pushed off screen when another tab is selected.

Reply:
Okay duh that makes sense. Thanks.